es6-promise/es6-promise.d.ts

```diff
@@ -51,6 +51,7 @@ declare module Promise {
 	 * Make a promise that rejects to obj. For consistency and debugging (eg stack traces), obj should be an instanceof Error
 	 */
 	function reject(error: any): Promise<any>;
+	function reject<T>(error: T): Promise<T>;
```

Comment:
@kant2002 This is now fixed on the TypeScript side: 
https://github.com/Microsoft/TypeScript/blob/master/lib/lib.es6.d.ts#L5455

Can we fix it here as well?